src/backend/opengl/CommandBufferGL.cpp

```diff
@@ -446,85 +374,249 @@ namespace backend { namespace opengl {
                     glDeleteFramebuffers(1, &readFBO);
                 } break;
 
+                case Command::TransitionBufferUsage: {
+                    TransitionBufferUsageCmd* cmd =
```

Comment:
Not new, but "auto" here would make this less rendundant, and make it wrap better.